content/en/dora_metrics/deployments/_index.md

```diff
@@ -33,98 +33,39 @@ The DORA Metrics private beta is closed. Fill out the form below to be added to
-  -H "DD-API-KEY: ${DD_API_KEY}" \
-  -d @- << EOF
-  {
-    "data": {
-      "attributes": {
-        "service": "shopist",
-        "started_at": 1693491974000000000,
-        "finished_at": 1693491984000000000,
-        "git": {
-          "commit_sha": "66adc9350f2cc9b250b69abddab733dd55e1a588",
-          "repository_url": "https://github.com/organization/example-repository"
-        },
-        "env": "prod"
-      }
-    }
-  }
-EOF
-```
-
-[1]: /api/latest/dora-metrics/#send-a-deployment-event-for-dora-metrics
-[2]: https://app.datadoghq.com/organization-settings/api-keys
-{{% /tab %}}
-
-{{% tab "datadog-ci CLI" %}}
-
-The [`datadog-ci`][1] CLI tool provides a shortcut to send deployment events within your Continuous Integration environment.
+## Calculating deployment frequency
 
-For the following example, set the `DD_SITE` environment variable to {{< region-param key="dd_site" code="true" >}} and set the `DD_API_KEY` environment variable to your [Datadog API Key][2]:
-```shell
-export DD_BETA_COMMANDS_ENABLED=1
-export DD_SITE="<DD_SITE>"
-export DD_API_KEY="<DD_API_KEY>"
+Deployment frequency is calculated based on the `dora.deployments.count` metric that is generated and increased with each deployment detected from your selected deployment data source. Frequency is calculated by dividing `dora.deployments.count` over a specific time frame.
 
-export deploy_start=`date +%s`
-./your-deploy-script.sh
-datadog-ci dora deployment --service shopist --env prod \
-    --started-at $deploy_start --finished-at `date +%s` \
-    --git-repository-url "https://github.com/organization/example-repository" \
-    --git-commit-sha 66adc9350f2cc9b250b69abddab733dd55e1a588
-```
-
-The deployment finish time is automatically set to now if `--finished-at` is not provided.
+## Calculating change lead time
 
-If the deployment CI job is running on the exact same Git revision that is being deployed, `git-repository-url` and `git-commit-sha` can be omitted and will be automatically inferred from the CI context.
-The `--skip-git` option can be provided to disable sending the repository URL and commit SHA. When this option is added, the Change Lead Time metric will not be available.
+For a single Git commit, the change lead time (CLT) is calculated as time from the creation of the commit to when the deployment including that commit was executed.
 
-[1]: https://www.npmjs.com/package/@datadog/datadog-ci
-[2]: https://app.datadoghq.com/organization-settings/api-keys
-{{% /tab %}}
-{{< /tabs >}}
+To calculate the change lead time for a deployment, Datadog runs [`git log`][6] between the deployment commit SHA and the previous deployment commit SHA to find all the commits being deployed. Then, it computes the average of the related change lead time values. Datadog doesn't store the actual content of files in your repository, only Git commit and tree objects.
 
-## Calculating Change Lead Time
+There are two requirements for calculating change lead time:
```

Comment:
I think we can remove this in favour of specifying everything needed in the different deployment sources pages. For example, point 1 is only important for API deployments. 

I would include in the APM deployments and API deployments a very similar sentence/setup, with the repository metadata synchronization which is in common (can both be duplicated or we could add a different page for that) and additional setup based on the source:

1. For API, deployments need to contain specific attributes
2. For APM, telemetry should be tagged with Git information

Reply:
IMO it makes sense to have a section per metric in this page so when the user skims it, they can tell from the headings which metrics deployment data is relevant for. Each section can provide brief summary for how deployments are used to calculate the metrics, and individual details can be left for the APM and API pages